inst/tests/test-formatter.r

```diff
@@ -53,6 +53,50 @@ test_that("dollar format", {
                  "$600,000", "$700,000", "$800,000", "$900,000", "$1,000,000"))
   expect_equal(dollar(c(9.999)), c("$10"))
   expect_equal(dollar(c(99.999)), c("$100"))
+  expect_equal(dollar(-c(100, 0.23, 1.456565, 2e3)),
```

Comment:
These tests seem a bit repetitive. Could you please reduce and break up into separately named tests?